access/src/main/java/edu/unc/lib/dl/ui/controller/FullRecordController.java

```diff
@@ -206,6 +206,32 @@ public String getFullRecord(String pid, Model model, HttpServletRequest request)
 			List<BriefObjectMetadataBean> neighbors = queryLayer.getNeighboringItems(briefObject,
 					searchSettings.maxNeighborResults, accessGroups);
 			model.addAttribute("neighborList", neighbors);
+			
+			// Get previous and next record in the same folder if there are any
+			List<BriefObjectMetadataBean> previousNext = new ArrayList<>();
+			
+			int selectedRecord = 0;
+			int index = 0;
+			for (BriefObjectMetadataBean neighbor : neighbors) {
```

Comment:
Can we use List#indexOf here? https://docs.oracle.com/javase/8/docs/api/java/util/List.html#indexOf-java.lang.Object-